azerothcore/azerothcore-wotlk
# fix(DB/Quest): "Botanist Taerix" and "Urgent Delivery!"

## Changes Proposed:
-  Add the correct previous quest requirements to Botanist Taerix and Urgent Delivery!


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4101



## Tests Performed:
- Ubuntu 20.04



## How to Test the Changes:

.go c id 16477
.q a 9369
.q c 9369
Select the Urgent Delivery! quest, complete it, now you can see another quest at Proenitus, Botanist Taerix.

Previously you could just pick up Botanist Taerix from the start.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
